WinUIGallery/ControlPages/DesignGuidance/IconsPage.xaml

```diff
@@ -92,235 +100,134 @@
         </DataTemplate>
     </Page.Resources>
 
-    <StackPanel Spacing="10" HorizontalAlignment="Stretch">
+    <Grid HorizontalAlignment="Stretch">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+        <StackPanel Orientation="Vertical">
         <RichTextBlock>
             <Paragraph>
-                With the release of Windows 11,
-                <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/style/segoe-fluent-icons-font">Segoe Fluent Icons Font</Hyperlink>
-                is the recommended symbol icon font.
-            </Paragraph>
+                With the release of Windows 11, <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/style/segoe-fluent-icons-font">Segoe Fluent Icons Font</Hyperlink>
+                is the recommended symbol icon font.</Paragraph>
         </RichTextBlock>
         <TextBlock Style="{ThemeResource BodyStrongTextBlockStyle}" Text="How do I get the font?" />
         <RichTextBlock>
             <Paragraph>
-                On Windows 11: There's nothing you need to do, the font comes with Windows.
-                <LineBreak/>
-                On Windows 10: The Segoe Fluent Icons is not included by default on Windows 10. You can download it
-                <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">here</Hyperlink> .
-                <LineBreak/>
+                On Windows 11: There's nothing you need to do, the font comes with Windows.<LineBreak />
+                On Windows 10: The Segoe Fluent Icons is not included by default on Windows 10. You can download it <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">here</Hyperlink>
+                .<LineBreak />
             </Paragraph>
         </RichTextBlock>
 
-        <TextBlock Style="{ThemeResource SubtitleTextBlockStyle}" Text="Fluent Icons Library" />
-        <Grid>
+        <TextBlock Style="{ThemeResource BodyStrongTextBlockStyle}" Text="Fluent Icons Library" />
+        <AutoSuggestBox
+            MinWidth="304"
+            MaxWidth="320"
+            HorizontalAlignment="Left"
+            PlaceholderText="Search icons"
+            QueryIcon="Find"
+            TextChanged="SearchTextBox_TextChanged" />
+        </StackPanel>
+        <Grid Grid.Row="1">
             <Grid.ColumnDefinitions>
-                <ColumnDefinition Width="*" MaxWidth="800"/>
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="296" />
             </Grid.ColumnDefinitions>
-            <Grid.RowDefinitions>
-                <RowDefinition Height="Auto" />
-                <RowDefinition Height="Auto" />
-            </Grid.RowDefinitions>
+            <ScrollViewer
+                Margin="-4,0,12,0"
```

Comment:
Please don't use negative margins.

Reply:
Have to.. to visually align the ItemsRepeater items with the rest of the content